src/index.js

```diff
@@ -0,0 +1,57 @@
+/*eslint-disable*/
+import _ from 'lodash';
+/* eslint-enable */
+import './style.css';
+
+const todolist = [
+  {
+    index: 1,
+    description: 'wash the dishes',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'clean my room',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: 'Complete todo list project',
+    completed: false,
+  },
+];
+class TODOs {
+  constructor() {
+    this.toDoList = [];
+  }
+
+  add(description, completed) {
+    this.toDoList = this.toDoList.concat({ index: Date.now(), description, completed });
+  }
+
+  remove(id) {
+    this.toDoList = this.toDoList.filter((todo) => todo.id !== Number(id));
+  }
+}
+/* eslint-disable */
+const todos = new TODOs();
+/* eslint-enable */
+
+const todoDiv = document.querySelector('.todos');
+const lunchTodoList = () => {
+  todolist.forEach((todo) => {
+    const li = document.createElement('li');
+    li.classList.add('todo');
+    // create checkbox
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = todo.completed;
+    li.appendChild(checkbox);
+    // create description
+    const desc = document.createElement('p');
+    desc.innerText = todo.description;
```

Comment:
- [ ] After the todo.description, you need to add a three dots icon, this is in the target design

Your design lacks the three dots icon:

![image](https://user-images.githubusercontent.com/60631456/125632635-8bce5a94-0035-4781-bc87-90209e6f4466.png)

Target design:

![image](https://user-images.githubusercontent.com/60631456/125632681-fbc6ab94-39a7-4401-b2e8-1c7f36920f8b.png)

Reply:
> * [ ]  After the todo.description, you need to add a three dots icon, this is in the target design
> 
> Your design lacks the three dots icon:
> 
> ![image](https://user-images.githubusercontent.com/60631456/125632635-8bce5a94-0035-4781-bc87-90209e6f4466.png)
> 
> Target design:
> 
> ![image](https://user-images.githubusercontent.com/60631456/125632681-fbc6ab94-39a7-4401-b2e8-1c7f36920f8b.png)

- [ ] As requested in the last review, please add the `three-dots-icon` as seen in the design.